assignments/callbacks.js

```diff
@@ -24,29 +24,92 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  return cb(arr.length);
 }
 
+getLength(items, function(length) {
+  console.log(length)
+});
+// answer ↑↑
+
+
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.pop());
 }
```

Comment:
YEAH buddy! I'm impressed that you thought to use pop instead of accessing it as an index value!